aidial_adapter_bedrock/app.py

```diff
@@ -16,14 +17,20 @@
 
 logging.config.dictConfig(LogConfig().dict())
 
-default_region = get_env("DEFAULT_REGION")
+DEFAULT_REGION = get_env("DEFAULT_REGION")
 
-app = DIALApp(description="AWS Bedrock adapter for DIAL API")
+OTLP_EXPORT_ENABLED: bool = (
+    os.environ.get("OTEL_EXPORTER_OTLP_TRACES_ENDPOINT") is not None
+)
 
-
-@app.get("/healthcheck")
-def healthcheck():
-    return fastapi.Response("OK")
+app = DIALApp(
+    description="AWS Bedrock adapter for DIAL API",
+    add_healthcheck=True,
+    telemetry_config=TelemetryConfig(
+        service_name="bedrock",
+        tracing=TracingConfig(oltp_export=OTLP_EXPORT_ENABLED),
```

Comment:
Looks like the name in the dial sdk should be fixed.

Reply:
Fixed